pages/releases.mdx

```diff
@@ -0,0 +1,149 @@
+import { Step } from '../components/Step'
+import { Callout, Steps } from "nextra/components"
+
+# Release notes
+*The latest Snaplet product updates from the Snaplet team.*
+
+
+<Steps>
+
+<Step>
+## Snaplet v0.72.0 Release Notes - 31 October 2023
```

Comment:
I would remove the "Release Notes" from the title, we're already in the release notes page.

Reply:
Good point!